db-random-test/src/test/scala/com/pingcap/tispark/test/RandomTestSuite.scala

```diff
@@ -43,12 +43,14 @@ class RandomTestSuite extends FunSuite with RandomTest {
     assert(rowCount == schemaAndData.data.size)
 
     val answer = """CREATE DATABASE IF NOT EXISTS `random_test`;
-                   |DROP TABLE IF EXISTS `random_test`.`test_351626634_1517918040`;
+                   |USE `random_test`;
+                   |DROP TABLE IF EXISTS `test_351626634_1517918040`;
                    |CREATE TABLE `random_test`.`test_351626634_1517918040` (
                    |  `col_int0` int(11) not null,
-                   |  `col_int1` int(11) not null /*T![clustered_index] CLUSTERED */
+                   |  `col_int1` int(11) not null
```

Comment:
Can you try verifying this with different TiDB versions. Clustered index was only available since TiDB 5.0.

we can use select version() to find out what version we are running testing against and make it works for different versions.

Reply:
+1

Reply:
This test is an unit test and has no relation with TiDB version. whether use Clustered index is hard code like this:
![image](https://user-images.githubusercontent.com/52435083/147739883-03b8a748-8a09-44e7-b904-f2ccf48585be.png)

The test errors because someone had moved Clustered index to primary key, but the test is not be changed.
![image](https://user-images.githubusercontent.com/52435083/147740425-0d0a8690-cd53-4805-8355-ca4a5965ec62.png)
![image](https://user-images.githubusercontent.com/52435083/147740602-c0bcfa33-f0ac-4e99-a067-7649b46d996a.png)

Reply:
> This test is an unit test and has no relation with TiDB version. whether use Clustered index is hard code like this: ![image](https://user-images.githubusercontent.com/52435083/147739883-03b8a748-8a09-44e7-b904-f2ccf48585be.png)
> 
> The test errors because someone had moved Clustered index to primary key, but the test is not be changed. ![image](https://user-images.githubusercontent.com/52435083/147740425-0d0a8690-cd53-4805-8355-ca4a5965ec62.png) ![image](https://user-images.githubusercontent.com/52435083/147740602-c0bcfa33-f0ac-4e99-a067-7649b46d996a.png)

I see. I thought they are integration tests.